package.json

```diff
@@ -78,7 +78,11 @@
 		"test-dangerci-and-adminpage-and-extensions": "yarn danger ci && yarn test-adminpage && yarn test-extensions",
 		"test-search": "jest modules/search",
 		"reformat-files": "./node_modules/.bin/prettier --ignore-path .eslintignore --write \"**/*.{js,jsx,json}\"",
-		"version-packages": "bash ./bin/version-packages.sh"
+		"version-packages": "bash ./bin/version-packages.sh",
+		"tunnel": "[ -f ./docker/bin/jt/tunnel.sh ] || { echo 'Use `yarn tunnel:install` to enable tunneling.'; exit; }; ./docker/bin/jt/tunnel.sh",
+		"tunnel:break": "[ -f ./docker/bin/jt/tunnel.sh ] || { echo 'Use `yarn tunnel:install` to enable tunneling.'; exit; }; ./docker/bin/jt/tunnel.sh break",
+		"tunnel:config": "[ -f ./docker/bin/jt/config.sh ] || { echo 'Use `yarn tunnel:install` to enable tunneling.'; exit; }; ./docker/bin/jt/config.sh",
+		"tunnel:install": "echo 'Auto-install is not available at the moment. Please visit https://github.com/Automattic/jurassic-tube for installation instructions.'"
```

Comment:
Since the repo is still private, I wonder if we should wait until the repo is public to add this option to the public Jetpack repo?